toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -532,7 +532,7 @@ func initializeRpmDatabase(installRoot string, diffDiskBuild bool) (err error) {
 			return err
 		}
 	}
-	err = initializeTdnfConfiguration(installRoot)
+	//err = initializeTdnfConfiguration(installRoot)
```

Comment:
Can you add a comment here about the issue this line causes and why it can be removed

Reply:
With the removal of this line I added some context to where `mariner-release` is added to the package list in the imager instead to add clarification as to why the change was made/needed.